src/bsoncxx/json.cpp

```diff
@@ -43,12 +43,28 @@ void bson_free_deleter(std::uint8_t* ptr) {
 
 }  // namespace
 
-std::string BSONCXX_CALL to_json(document::view view) {
+enum class jsonMode { LEGACY, CANONICAL, RELAXED };
```

Comment:
- If this is only used in this file, it belongs in the unnamed namespace
- Types start with capital letters
- Enum constants start with k, so kLegacy, etc.

Reply:
- ALL_CAPS_IS_SHOUTY_AND_UNECESSARY_IN_CXX: https://accu.org/index.php/journals/1923
- The k prefix comes from the Google Style Guide and is consistently followed throughout the codebase. Taking another look, it should really be `k_legacy`, etc. But presumably we aren't keeping them.